extras/manifests/prometheus-ceph-rules.yaml

```diff
@@ -56,13 +69,13 @@ spec:
     - alert: CephMonHighNumberOfLeaderChanges
       annotations:
         description: 'Ceph Monitor "{{ $labels.job }}": instance {{ $labels.instance
-          }} has seen {{ $value }} leader changes recently.'
+          }} has seen {{ $value }} leader changes per minute recently.'
```

Comment:
Not a part of this PR.

Reply:
May be it was not generated and pushed as part of earlier PR so comes here.

Reply:
Should I send another PR to correct the same?